sycl/doc/extensions/proposed/sycl_ext_oneapi_complex_algorithms.asciidoc

```diff
@@ -50,6 +52,13 @@ John Pennycook, Intel (john 'dot' pennycook 'at' intel 'dot' com)
 
 This extension is written against the SYCL 2020 specification, Revision 4.
 
+This extension builds on the `sycl::ext::oneapi::complex` and
+`marray<sycl::ext::oneapi::complex>` classes. Therefore this extension is
+dependent on link:sycl_ext_oneapi_complex.asciidoc[sycl_ext_oneapi_complex]
+and
+link:sycl_ext_oneapi_complex_marray.asciidoc[sycl_ext_oneapi_complex_marray].
```

Comment:
Prior to this change, developers could use the `std::complex` features without requiring those other two extensions.  Does anything in our `std::complex` support now depend on these new extensions?  If not, I think we should be explicit that only those parts of this extension which use `sycl::complex` or `marray` are dependent.